examples/progressbar.py

```diff
@@ -1,30 +1,21 @@
-import time
+from time import sleep
 
```

Comment:
Is this file a gif? 

Or you must send two pull requests with separate title.

Reply:
No, you are right, I will be sending another pull request for it.